### ISSUE TITLE
Typo in Fleet Install docs

### ISSUE THREAD

**Issue by ejether** [FairwindsOps/insights-docs#137]
https://github.com/FairwindsOps/insights-docs/blob/a5ac4c4b9b6c9b648e74dda9b301122c582c6291/docs/run/agent/installation.md?plain=1#L38

Should match expected values path:
https://github.com/FairwindsOps/charts/blob/master/stable/insights-agent/templates/fleet-installer/admin-secret.yaml#L5

Or should be removed from the docs so it can fallback to the default

`insights.tokenSecretName != insights.apiTokenSecretName`

👍

**Comment by rbren** [comment]
🤦 thanks EJ!

**Comment by ejether** [comment]
I'm seeing that the `tokenSecretName` is required and everywhere... https://github.com/FairwindsOps/charts/search?q=tokenSecretName


I may have been hasty in some assumptions but I know I'm getting a ```Warning  Failed     6s (x5 over 51s)  kubelet            Error: secret "insights-agent-api-token" not found``` when I follow the fleet instructions as written

**Comment by ejether** [comment]
Looks like its actually coming from here: https://github.com/FairwindsOps/charts/blob/a38ce4043808ec562fc7757dd858019dda358c94/stable/insights-agent/templates/token-secret.yaml

**Comment by rbren** [comment]
Sorry for the confusion!

`tokenSecretName` is for the _cluster_ token. `apiTokenSecretName` is for the _api_ token, which will be used to configure the cluster token if specified.

I believe the docs should work now that I've changed `tokenSecretName` to `apiTokenSecretName`

**Comment by ejether** [comment]
I did try that and I got a: ` Error: execution error at (insights-agent/templates/fleet-installer/job.yaml:51:28): You must specify insights.tokenSecretName for a fleet install`

Clearly, I'm just scratching the surface here so I'm probably missing some connections.

**Comment by rbren** [comment]
🤔 maybe the docs were correct.

👀

**Comment by rbren** [comment]
Yeah docs were correct as they were initially. Did that not work for you?

What is supposed to happen is:
* a secret named `apiTokenSecretName` is created (with a sensible default for that name)
* a Job kicks off that
  * creates a new cluster in the Insights API
  * downloads that cluster's token
  * creates a Secret named `tokenSecretName` with the cluster's token

**Comment by ejether** [comment]
When I followed the docs, I got `Warning Failed 6s (x5 over 51s) kubelet Error: secret "insights-agent-api-token" not found` By setting `tokenSecretName: insights-agent-api-token` it seems to work around it.

When I set `tokenSecretName` to "insights-agent-api-token", it seems to work. 

My _assumption_ is that this secret: https://github.com/FairwindsOps/charts/blob/a38ce4043808ec562fc7757dd858019dda358c94/stable/insights-agent/templates/fleet-installer/admin-secret.yaml#L5

which has `apiTokenSecretName` as it's default is getting named `insights-agent-api-token` as a default and I'd (maybe) need to align `apiTokenSecretName` and `tokenSecretName`

```yaml
 values:
  - fleetInstall: true
    insights:
      organization: "openraven"
      cluster: {{ $cleanClusterName }}
      tokenSecretName: insights-agent-api-token
      apiToken: {{ $decryptedToken }}
```


Also, I apologize if I'm just being dumb and overlooking something in the docs.
